nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -130,42 +134,85 @@ where
-                    supply: Set(1),
-                    supply_mint: Set(None),
-                    compressed: Set(true),
                     tree_id: Set(Some(bundle.keys.get(3).unwrap().0.to_vec())),
-                    specification_version: Set(SpecificationVersions::V1),
-                    specification_asset_class: Set(SpecificationAssetClass::Nft),
-                    nonce: Set(nonce as i64),
-                    leaf: Set(Some(le.leaf_hash.to_vec())),
+                    specification_version: Set(Some(SpecificationVersions::V1)),
+                    specification_asset_class: Set(Some(SpecificationAssetClass::Nft)),
+                    nonce: Set(Some(nonce as i64)),
                     royalty_target_type: Set(RoyaltyTargetType::Creators),
                     royalty_target: Set(None),
                     royalty_amount: Set(metadata.seller_fee_basis_points as i32), //basis points
                     asset_data: Set(Some(id_bytes.to_vec())),
-                    seq: Set(seq as i64), // gummyroll seq
-                    slot_updated: Set(slot_i),
+                    slot_updated: Set(Some(slot_i)),
                     data_hash: Set(Some(data_hash)),
                     creator_hash: Set(Some(creator_hash)),
                     ..Default::default()
                 };
 
-                // Do not attempt to modify any existing values:
-                // `ON CONFLICT ('id') DO NOTHING`.
-                let query = asset::Entity::insert(model)
+                // Upsert asset table base info.
+                let query = asset::Entity::insert(asset_model)
                     .on_conflict(
                         OnConflict::columns([asset::Column::Id])
-                            .do_nothing()
+                            .update_columns([
+                                asset::Column::OwnerType,
+                                asset::Column::Frozen,
+                                asset::Column::TreeId,
+                                asset::Column::SpecificationVersion,
+                                asset::Column::SpecificationAssetClass,
+                                asset::Column::Nonce,
+                                asset::Column::RoyaltyTargetType,
+                                asset::Column::RoyaltyTarget,
+                                asset::Column::RoyaltyAmount,
+                                asset::Column::AssetData,
+                                //TODO maybe handle slot updated differently.
+                                asset::Column::SlotUpdated,
+                                asset::Column::DataHash,
+                                asset::Column::CreatorHash,
+                            ])
                             .to_owned(),
                     )
                     .build(DbBackend::Postgres);
                 txn.execute(query).await?;
 
+                // Partial update of asset table with just compression info elements.
```

Comment:
Breaking this apart into multiple upserts will put more load on the DB. Did you an analysis on the DB load changes? Did you test this under high load (e.g. 1kTPS+)?

Reply:
So overall for this PR I just need to determine final strategy to complete it.  Here are the options I see right now based on these comments:

1. Just test it under load with the multiple upserts as-is, and if its working fine under high load, leave it as-is.
2. Modify the code to have one upsert followed by 1-2 updates for each program operation.  I initially chose all upserts because it was simpler code and less prone to issue if someone changes order of things, but updates will be less load than upserts. 
3. Modify the code to just use the `seq` for `leaf` and `owner_delegate_seq` for `owner`+`delegate`.  This removes one of the upserts/updates from each operation.

Reply:
I would test it under load beforehand and then see how it holds up under load afterwards. Especially curious about memory stress. We have been running into issues with a 256GB RAM DB when we're under higher loads.

I'll be carving out some time to do a deeper analysis to figure out the conditions that specifically cause the pain. I believe its mostly due to the high number of indexes and updating all of them quickly is expensive on memory. Especially as the cl_items table grows quite large.

Reply:
Tested by Triton on their non-prod environment.  Also Helius doing some testing.  Marking as resolved.